### DISCUSSION TITLE
Subscription sharing for "Private Username Mode" users.

### DISCUSSION THREAD

**Discussion by MichaelDevon** [standardnotes/forum#3386] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

Yes.

**Is your feature request related to a problem? Please describe.**

I can't use the professional plan's account sharing feature with users without email addresses attached to their accounts. 

**Describe the solution you'd like**

Accept the long string that is generated by the username as an "email", in the field, and add the accounts to the sharing pool automatically if the account exists, since typos are impossible this way, theres no need to send out any email.

**Describe alternatives you've considered**

none

**Additional context**

This makes the account sharing feature useless to me, and this is the only feature i paid extra for in the PRO plan compared to the productivity. I think this is not very hard to implement, and if the team decides not to implement it, since i read that you guys reject most "feature requests", make it very clear in the features overview that this is not supported, as i wasted extra money for stuff i cant use.

**Comment by effieeee** [comment]
Hi there, I do understand your frustration, but I'm afraid I can't guarantee that a solution for subscription sharing on Private Usernames will be made any time soon. In case you purchased your subscription fairly recently, we can definitely provide a refund for you in light of that. Please send us an email about this at help@standardnotes.com 🙏

**Reply by MichaelDevon** [reply]
Hello! Is there a specific reason why it wont be implemented? 
I think its very important to keep the app as easy to recommend for privacy as possible. 
Amongst many other things such as independent audits, the app has this killer feature (username mode) that i think the customers who pay the extra for the premium would expect to work seamlessly. Also, i may not know much about development, but i think its not as hard to implement than an entirely new feature. 

Can my friend add an email to their account after it was created with username only, or is it considered permanently crippled regarding using this feature?

**Reply by MichaelDevon** [reply]
Also, is this not an issue the dev team will work on in the future any time ever? I think keeping the issue open, instead of making it a discussion would be productive, if you think its somewhat important to fix.

**Reply by effieeee** [reply]
Hey Michael, adding an email to the account after it was created via username only kind of defeats the purpose of the private username functionality, so I'm afraid I wouldn't suggest that as a workaround either 😞 As far as I'm aware, the subscription sharing function also uses the emails as an added verification step, though if there are also any other technical reasons for why sharing isn't available for private usernames, I'm afraid I don't have those details at the moment.

Like most [feature requests](https://standardnotes.com/longevity), I'm afraid we can't guarantee any action on it unless we explicitly say that it's coming soon. In terms of where this should be on the Forum though, it's better placed under the **Suggestions** section here, since the Issues page is better used to track malfunctions on the app and its features 🙏
